### ISSUE TITLE
PLOGO - Pascal Logo

### ISSUE THREAD

**Issue by larsbrinkhoff** [PDP-10/its#1302]
We have blessing from Gary Drescher to publish PLOGO.

Need a Pascal compiler first, see #505.

**Comment by larsbrinkhoff** [comment]
Related to this, add the Pascal mode for EMACS.

**Comment by larsbrinkhoff** [comment]
Hello @leighklotz,

This is the GitHub issue for researching PLogo and add it to the ITS repository.

**Comment by larsbrinkhoff** [comment]
Neither of SYS2; TS PASCAL or PASCAL; TS PASCAL accepts this source file.

**Comment by leighklotz** [comment]
> Neither of SYS2; TS PASCAL or PASCAL; TS PASCAL accepts this source file.
>
> --
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/PDP-10/its/issues/1302#issuecomment-427752957

Yeah, I don't know what it was compiled on.  It was written before the TI
9900 development machine arrived and when compiled it turned into about
400Kb, way above the size of the 99/4, so Ed Hardebeck hand-compiled it
into 9900 assembler.  Steve Hain did the same (as a side project) for the
6502, as a student.  Patrick Sobalvarro and I got it working and Patrick
wrote the editor.  I finished the project, which took about 2 more years 
and then moved with it to Terrapin, and also Leona Educational Systems in
Tokyo (where I helped port it to 6809).

I never saw plogo run, though I occasionally looked at it, because it was
the only documentation for the 6502 code, which was largely uncommented.

**Comment by larsbrinkhoff** [comment]
I will try to get these files:

PLOGO; TS PASCAL  
PLOGO; TS PCROSS  
PLOGO; PASCAL PAS  
PLOGO; PCROSS PAS
